src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -90,8 +90,20 @@ def _delay(self):
 
     def __call__(self, poller):
         logger.info("Starting long running operation '%s'", self.start_msg)
+        correlation_message = ''
         while not poller.done():
-            self._delay()
+            try:
+                correlation_message = 'Correlation ID: {}' \
+                    .format(json.loads(poller._response.__dict__['_content']) #pylint: disable=protected-access
+                            ['properties']['correlationId'])
```

Comment:
yes, this is the only name I've seen and it is consistent.  The operation ID isn't available in the response, so I am only grabbing this one

---
In reply to: [80130661](https://github.com/Azure/azure-cli/pull/957#discussion_r80130661) [](ancestors = 80130661)